### DISCUSSION TITLE
Does Apache Skywalking Zabbix integration support TLS or PSK encryption?

### DISCUSSION THREAD

**Discussion by roman-yaryzhenko** [apache/skywalking#11463] [Q&A]
I'm in process of monitoring system selection and Apache Skywalking sees very interesting for me because of Zabbix Agent support (it was my favourite instrument in past), but does Skywalking support any encryption mechanisms from Zabbix Agent?

**Comment by mrproliu** [comment]
No, we only support the plain text.

**Comment by wu-sheng** [comment]
If it is a standard TLS, you could put a proxy like Envoy in front of OAP. Then you could forward plain to us and encrypted on connection.
